app/controllers/application_controller.rb

```diff
@@ -12,6 +12,10 @@ class ApplicationController < ActionController::Base
 
   layout proc { kind_of?(Europeana::Styleguide) ? false : 'application' }
 
+  rescue_from ActiveRecord::RecordNotFound, ActionController::RoutingError do |exception|
```

Comment:
Unused block argument - `exception`. You can omit the argument if you don't care about it.